packages/hash/api/src/model/ontology/entity-type.model.ts

```diff
@@ -64,13 +65,15 @@ export default class {
   static async get(
     graphApi: GraphApi,
     params: {
-      accountId: string;
       versionedUri: string;
     },
   ): Promise<EntityTypeModel> {
-    const { accountId, versionedUri } = params;
+    const { versionedUri } = params;
     const { data: schema } = await graphApi.getEntityType(versionedUri);
 
+    /** @todo: retrieve accountId from `graphApi.getEntityType` response */
+    const accountId = NIL_UUID;
+
     return new EntityTypeModel({ schema, accountId });
```

Comment:
@Alfred-Mountfield @TimDiekmann do we have an existing ticket for this?

Reply:
https://app.asana.com/0/1201095311341924/1202770062090893/f

Reply:
Danke :)